validator-api/src/cache.rs

```diff
@@ -0,0 +1,34 @@
+use anyhow::Result;
+use mixnet_contract::MixNodeBond;
+use std::time::Instant;
+use validator_client::Client;
+
+pub struct MixNodeCache {
+    value: Vec<MixNodeBond>,
+    as_at: Instant,
```

Comment:
Hm, just a suggestion, how about changing `as_at` from `Instant` to a `u64` value, where that value would be the height of the chain when we grabbed that information? We can get it from, for example in the case of our validator `https://testnet-finney-validator.nymtech.net/block` (if we don't specify arguments for block, we get the latest one, and then we can grab it from `block.header.height`)

Reply:
Also, temporary for linter sake, just slap a `#[allow(dead_code)]` on top of it.

Reply:
I'd prefer to leave `as_at` to be time, so that we can invalidate cache based on time if we need to, I do like adding the block height